docs/tables/aws_cloudtrail_trail_event.md

```diff
@@ -38,13 +39,52 @@ select
 from
   aws_cloudtrail_trail_event
 where
-  log_group_name = 'aws-cloudtrail-logs-013122550996-77246e11' and
-  not read_only
+  log_group_name = 'required-your-log-group-name'
+  and timestamp >= (now() - interval '5' minute);
```

Comment:
```suggestion
  and timestamp >= now() - interval '5 minute';
```

As per https://github.com/turbot/steampipe-plugin-aws/pull/1136#issuecomment-1176671964, can you please make the change above and to other examples as well?

Reply:
Fixed